test/Typewriter.Test/Given_a_valid_metadata_file_to_Typewriter.cs

```diff
@@ -555,5 +555,145 @@ public void It_transforms_metadata()
             Assert.IsTrue(hasContainsTargetBeenSet, $"The expected ContainsTarget attribute wasn't set in the transformed cleaned metadata.");
             Assert.IsFalse(hasCapabilityAnnotations, $"The expected capability annotations weren't removed in the transformed cleaned metadata.");
         }
+
+        [Test, RunInApplicationDomain]
```

Comment:
nit: tests can be combined with data attributes.

Reply:
Fixed with 405c1861

Reply:
yay, less code! :) Thanks for taking the time to make these changes. Really appreciate it.